### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by sabine** [ocaml/ocaml.org#880]
- [ ] improve color scheme to meet contrast guidelines
- [ ] use semantic HTML to render all navigation elements and mark them appropriately
- [ ] https://github.com/ocaml/ocaml.org/issues/829

**Comment by sabine** [comment]
@shindere I recently overhauled the main navbar to use more semantic HTML (https://github.com/ocaml/ocaml.org/pull/851) and, thus, it is hopefully more accessible. Always very interested in feedback on individual accessibility items like this. Or we schedule to meet for comprehensive user testing all across the site end of this month or next month. I've been trying to test a bit with the built-in screen reader in Ubuntu, and it's been insightful, but I cannot really judge the quality of the user experience because I think I need to learn a few keyboard shortcuts that anyone experienced with a screen reader probably uses daily.

**Comment by MelakuAlehegn** [comment]
Hi @sabine @SaySayo Can I be assigned for this so that I can check accessibility and best practices? if there are any?

**Comment by MelakuAlehegn** [comment]
@sabine @SaySayo any answers to this please?

**Comment by tmattio** [comment]
We've redesigned most pages since this issue was open.

**Comment by shindere** [comment]
Thibaut Mattio (2024/10/01 02:00 -0700):
> We've redesigned most pages since this issue was open.

Thanks. I am not convinced the issue can be closed as completed, though,
because the re-designed pages should at the very minimum be tested by
people with different disabilitiesand ideally undergo a proper
accessibility audit.

My suggestion would thus be to re-open this issue to track the fact
that, unless I am missing information, neither testing nor auditing has
happened so far.

**Comment by sabine** [comment]
Indeed, this hasn't been checked for the new pages.
